docs/book/src/tasks/upgrading-clusters.md

```diff
@@ -0,0 +1,66 @@
+# Upgrading management and workload clusters
+
+## Considerations
+
+### Supported versions of Kubernetes
+
+If you are upgrading the version of Kubernetes for a cluster managed by Cluster API, check that the [running version of
+Cluster API on the Management Cluster supports the target Kubernetes version](../reference/versions.md).
+
+You may need to [upgrade the version of Cluster API](upgrading-cluster-api-versions.md) in order to support the target
+Kubernetes version.
+
+In addition, you must always upgrade between Kubernetes minor versions in sequence, e.g. if you need to upgrade from
+Kubernetes v1.17 to v1.19, you must first upgrade to v1.18.
+
+
+### Images
+
+For kubeadm based clusters, infrastructure providers require a "machine image" containing pre-installed, matching
+versions of `kubeadm` and `kubelet`, ensure that relevant infrastructure machine templates reference the appropriate
+image for the Kubernetes version.
+
+## Upgrading using Cluster API
+
+The high level steps to fully upgrading a cluster are to first upgrade the control plane and then upgrade
+the worker machines.
+
+### Upgrading the control plane machines
+
+#### How to upgrade the underlying machine image
+
+To upgrade the control plane machines underlying machine images, the `MachineTemplate` resource referenced by the
+`KubeadmControlPlane` must be changed. Since `MachineTemplate` resources are immutable, the recommended approach is to
+
+1. Copy the existing `MachineTemplate`.
+2. Modify the values that need changing, such as instance type or image ID.
+3. Create the new `MachineTemplate` on the management cluster.
+4. Modify the existing `KubeadmControlPlane` resource to reference the new `MachineTemplate` resource in the `infrastructureRef` field.
+
+The final step will trigger a rolling update of the control plane using the new values found in the `MachineTemplate`.
+
+#### How to upgrade the Kubernetes control plane version
+
+To upgrade the Kubernetes control plane version, which will likely, depending on the provider, also upgrade the
+underlying machine image, make a modification to the `KubeadmControlPlane` resource's `Spec.Version` field. This will
+trigger a rolling upgrade of the control plane.
```

Comment:
```suggestion
To upgrade the Kubernetes control plane version make a modification to the `KubeadmControlPlane` resource's `Spec.Version` field. This will trigger a rolling upgrade of the control plane and, depending on the provider, also upgrade the underlying machine image. 
```